controllers/activemqartemis_controller_deploy_operator_test.go

```diff
@@ -34,9 +36,27 @@ import (
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const dummyPythonScript = `
+import signal, sys; signal.signal(signal.SIGTERM, lambda signum, frame: sys.exit(0));
+from http.server import SimpleHTTPRequestHandler, HTTPServer
+class DummyHandler(SimpleHTTPRequestHandler):
+	def do_GET(self):
+		response_content = '{\"request\":{\"mbean\":\"org.apache.activemq.artemis:broker=\\\"0.0.0.0\\\"\",\"attribute\":\"Status\",\"type\":\"read\"},\"value\":\"{\\\"configuration\\\":{\\\"properties\\\":{\\\"broker.properties\\\":{\\\"alder32\\\":\\\"1\\\",\\\"fileAlder32\\\":\\\"1285359230\\\"}}},\\\"server\\\":{\\\"jaas\\\":{\\\"properties\\\":{}},\\\"state\\\":\\\"STARTED\\\",\\\"version\\\":\\\"2.42.0\\\",\\\"nodeId\\\":\\\"27d63164-aa5b-11f0-a502-cabd4cd943ed\\\",\\\"identity\\\":null,\\\"uptime\\\":\\\"1 minute\\\"}}\",\"status\":200}'.encode('utf-8');
+		self.send_response(200);
+		self.send_header('Content-type', 'application/json');
+		self.send_header('Content-Length', str(len(response_content)));
+		self.end_headers();
+		self.wfile.write(response_content);
+HTTPServer(('', 8161), DummyHandler).serve_forever()
```

Comment:
I'm wondering if this shouldn't live in its own file instead, if we want to make improvements to the dummy broker in the future, it'll be easier to find.